.github/workflows/ci.yml

```diff
@@ -17,20 +19,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - '1'
-        os:  [ubuntu-latest, windows-latest, macos-latest]
-        arch:
-          - x64
+        version: [1.6, 1]
```

Comment:
I would prefer if this is left as only version `1`. There is no reason to test two versions, latest stable Julia is more than enough. (I am very climate consicous and I believe we all should be avoiding computations that are not necessary)